lightning/src/onion_message/offers.rs

```diff
@@ -81,6 +82,22 @@ impl OffersMessage {
 	}
 }
 
+impl fmt::Debug for OffersMessage {
+	fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+		match self {
+			OffersMessage::InvoiceRequest(message) => {
+				write!(f, "InvoiceRequest: {:?}", message.as_tlv_stream())
+			}
+			OffersMessage::Invoice(message) => {
+				write!(f, "Invoice: {:?}", message.as_tlv_stream())
+			}
+			OffersMessage::InvoiceError(message) => {
+				write!(f, "InvoiceError: {:?}", message)
+			}
```

Comment:
We can drop the string prefixes as the `Debug` of each type will include the type name already.